project.clj

```diff
@@ -40,8 +41,8 @@
                  ;; we have documentation that shows how users can use it to
                  ;; send their logs to logstash, so we include it in the jar.
                  ;; we may use it directly in the future
-                 [net.logstash.logback/logstash-logback-encoder "4.5.1"]
-
+                 [net.logstash.logback/logstash-logback-encoder "4.5.1" :exclusions [ch.qos.logback/logback-access
+                                                                                     ch.qos.logback/logback-core]]
```

Comment:
I suggested this approach because we are trying to manage the logback version via the versions specified in the Trapperkeeper project.clj.  Otherwise we risk ending up with different versions of some of the 3 logback artifacts, and when that happens, you are subject to weird AbstractMethod errors that aren't always caught by our tests (we shipped a broken release once due to this exact issue).

@jpinsonault maybe it wouldn't hurt to add a comment here, though, saying something like "logback versions should be inherited from trapperkeeper".